### ISSUE TITLE
Ordering the Filters

### ISSUE THREAD

**Issue by ShayneJud** [volkanceylan/Serenity#4753]
Is there an "easy/recommended" way to change the Order the QuickFilters (or any other Filters) appear above the Grid? An Attribute or other method? (Without changing their order in the Grid, of course...)

**Comment by ShayneJud** [comment]
If no "Attribute", I suppose I can use getQuickFilters() and  just pop them off the array and add them back in the order I want. Assuming they are rendered from that Array. (It appears the "element" contains the rendering HTML....)

**Comment by ShayneJud** [comment]
In case anyone is interested, here is how I handled this.

In Grid.ts, at the end of getQuickFilters() (I was already adding some custom filters...) I added this:

 filters.sort((x, y) => { return x.field == fld.ShipDate ? -1 : y == fld.ShipDate ? 1 : 0; });

This also requires "import fld = vwCMSSalesByResourceRow.Fields;" (above the class declaration).

"ShipDate" is the field I wanted to appear first in the filters.
